Tribler/Core/DownloadConfig.py

```diff
@@ -16,13 +16,16 @@
 import logging
 
 import os
+from ConfigParser import ParsingError, MissingSectionHeaderError
 from types import StringType
 
 from Tribler.Core.Utilities.configparser import CallbackConfigParser
 from Tribler.Core.defaults import dldefaults
 from Tribler.Core.osutils import get_home_dir
 from Tribler.Core.simpledefs import DLMODE_VOD, STATEDIR_DLCONFIG, UPLOAD
 
+logger = logging.getLogger(__name__)
```

Comment:
Maybe it's a bit cleaner to make this a class variable of `DownloadConfigInterface`?